app/src/lib/tokens/Pairing.svelte

```diff
@@ -3,38 +3,34 @@
 	import Loading from './Loading.svelte'
 	import NotFound from './NotFound.svelte'
 	import Result from './Result.svelte'
-	import Token from './Token.svelte'
+	import TokenDisplay from './TokenDisplay.svelte'
 	import {check_ontology} from '$lib/lookups'
 	import {REGEXES} from '$lib/regexes'
+	import {create_error_token} from '$lib/parser/token'
 
-	/** @type {CheckedToken} */
-	export let checked_token
+	/** @type {Token} */
+	export let token
 
-	$: [left, right] = split_pair(checked_token.token)
+	// At this point the pairing tokens will never be null. This satisfies the compiler
```

Comment:
maybe a `// ts-ignore` instead of the extra code?

Reply:
Good call